scripts/cart.js

```diff
@@ -0,0 +1,54 @@
+/* Funtion Sumar input*/
+function sumar() {
+    let inputElement = document.querySelector('.js-product-add-card-number');
+    let valorActual = parseInt(inputElement.value);
+    inputElement.value = valorActual + 1;
+  }
```

Comment:
Crei que esto era una funcion gigante con funciones dentro. Cuidado con la indentacion.
```suggestion
function sumar() {
  let inputElement = document.querySelector('.js-product-add-card-number');
  let valorActual = parseInt(inputElement.value);
  inputElement.value = valorActual + 1;
}
```